packages/cloud-cognitive/src/index-without-carbon-released-only.scss

```diff
@@ -5,8 +5,7 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@import './global/styles/imported-carbon-modules';
```

Comment:
@dbrugger Would you know if this import path is still being referenced in Cloud Pak for Security? If so, I'm assuming removing it may cause Sass to fail to compile.